src/libraries/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/ComInterop/ComRuntimeHelpers.cs

```diff
@@ -179,7 +179,7 @@ internal static ComTypes.TYPEATTR GetTypeAttrForTypeInfo(ComTypes.ITypeInfo type
 
             try
             {
-                return (ComTypes.TYPEATTR)Marshal.PtrToStructure(pAttrs, typeof(ComTypes.TYPEATTR));
+                return Marshal.PtrToStructure<ComTypes.TYPEATTR>(pAttrs);
```

Comment:
```suggestion
                return *(ComTypes.TYPEATTR*)(pAttrs);
```
This should just a simple pointer dereference. It is unnecessary to call PtrToStructure here.